### ISSUE TITLE
Docker image with `jbt` installed

### ISSUE THREAD

**Issue by Danil42Russia** [JetBrains/JetBrainsRuntime#420]
It would be handy if there was a ready-made Docker image with `jbr` pre-installed. Yes, you can build it yourself, but then you have to build the infrastructure for it.

How others do it:
- OpenJDK: https://hub.docker.com/_/openjdk
-

**Comment by mkartashev** [comment]
Yes, we only have [build environments published](https://hub.docker.com/r/jetbrains/runtime/tags), but not JBR as a build result. Worth considering.

**Comment by mkartashev** [comment]
It makes sense to publish an image per CPU (Critical Patch Update) of the corresponding OpenJDK version. Would that be acceptable?
